superset-frontend/cccs-viz/cccs_create_dev_config.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# determine directory of this script
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+CONFIG_FILE=$DIR/../../docker/pythonpath_dev/superset_config_docker.py
+
+read -p 'Enter path hogwarts-superset-deployment/config/sql_query_mutator.py file: ' SQL_QUERY_MUTATOR_FILE
+
+echo "Adding $SQL_QUERY_MUTATOR_FILE content"
+echo "to $CONFIG_FILE"
+cat $SQL_QUERY_MUTATOR_FILE > $CONFIG_FILE
+
+echo "Adding dev configs"
+cat >> $CONFIG_FILE <<'_EOF'
+SQL_QUERY_MUTATOR = sql_query_mutator 
+SUPERSET_WEBSERVER_TIMEOUT = 100
```

Comment:
Is this a deliberate increase from the default? Were you running into timeouts? Just curious.

Reply:
I was not running into timeouts, the default is 60?

Reply:
Yeah I believe so. Could just leave this line out if the default's good, but not a big deal.